### ISSUE TITLE
Feature request - User details API

### ISSUE THREAD

**Issue by rachit2427** [cowinapi/developer.cowin#146]
Hi,

An API to fetch user data like -
- Vaccination status (unvaccinated, partial, full)
- Number of doses taken
- Vaccine name
- Vaccination date

would be useful. Currently, we can only fetch the user certificate which is in PDF format. To develop an app, like a vaccine tracker, it's not feasible to parse PDFs. An API that returns a well-accepted format, like JSON, should help developers build apps much more efficiently.

Is there any possibility of such API?

cc: @cowinapi @satishchandra-pandey @sudhir-trigyn

**Comment by Anirudhk07** [comment]
@rachit2427 Absolutely, this API is the need of the hour. #100

**Comment by souvik234** [comment]
Since there is already a beneficiary schema built in with a unique ID, this should be fairly easy to implement, and can add useful functionality.

**Comment by Anirudhk07** [comment]
> Since there is already a beneficiary schema built in with a unique ID, this should be fairly easy to implement, and can add useful functionality.

But the beneficiary schema is not Public @souvik234

**Comment by souvik234** [comment]
@Anirudhk07 Yes, I'm aware. What I meant was that since the difficult work of designing a schema is already done, its easy now to open it up, and it won't require considerable effort.

**Comment by Anirudhk07** [comment]
> @Anirudhk07 Yes, I'm aware. What I meant was that since the difficult work of designing a schema is already done, its easy now to open it up, and it won't require considerable effort.

Oh right!
